components/lsp/MintForm.tsx

```diff
@@ -73,6 +76,14 @@ const MintForm: FC<Props> = ({
     }
   }, [signer, setShowWallet]);
 
+  useEffect(() => {
+    if (amount && toWeiSafe(amount).gt(collateralBalance)) {
```

Comment:
i think the decimals need to be taken into consideration for non 18 collaterals

Reply:
Done!